### PR TITLE
Add Angular 2+ icon, link

### PR DESCRIPTION
**Pull request by frntnd93**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Enhancement
- [ ] Documentation Update

## Description
Adds [Angular 2+](https://angular.io/) as a skill under Frontend Development and website link.
## Related Tickets & Documents
Related to #373 
## QA Instructions, Screenshots, Recordings
image

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] readme

### CODE REVIEW

File: src/constants/skills.js [hunk 1]
```diff
@@ -221,6 +222,8 @@ const icons = {
     "https://raw.githubusercontent.com/devicons/devicon/master/icons/react/react-original-wordmark.svg",
   angularjs:
     "https://raw.githubusercontent.com/devicons/devicon/master/icons/angularjs/angularjs-original-wordmark.svg",
+  angular:
+    "https://raw.githubusercontent.com/devicons/devicon/master/icons/angularjs/angularjs-plain.svg",
```
Comment (aravindvnair99) [inline src/constants/skills.js:226]:
```suggestion
    "https://angular.io/assets/images/logos/angular/angular.svg",
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@frntnd93 [AngularJS](https://angularjs.org/) logo is different from [Angular](https://angular.io/) logo.

**Review by aravindvnair99 — APPROVED** [review]

**Comment by frntnd93** [comment]
> @frntnd93 [AngularJS](https://angularjs.org/) logo is different from [Angular](https://angular.io/) logo.

You're right, commit suggestion applied

**Review by harshcut — APPROVED** [review]
LGTM 👍

**Pull request merged**